src/Dropdown.vue

```diff
@@ -45,6 +53,12 @@
         required: false,
         default: false,
         note: 'Disable the dropdown'
+      },
+      maxItem: {
+        type: Number,
+        required: false,
+        default: 10,
```

Comment:
You should change the default to 6 to keep it consistent with previous version and avoid having a scrollbar by default 
```suggestion
        default: 6,
```

Reply:
no probs